### ISSUE TITLE
enforce-consistent-line-wrapping: printWidth is compared against total raw string length instead of the longest individual line

### ISSUE THREAD

**Issue by fkoepke** [sergioazoc/oxlint-tailwindcss#110]
The rule compares the **total raw length** of the class string (including embedded newlines and indentation) against `printWidth`. This makes it impossible to satisfy the rule by splitting into multiple lines, because every newline and indentation character added by wrapping only makes the raw string longer.

The following multiline convention is affected:

```tsx
<div className={`
  flex items-center justify-between
  bg-white rounded-lg shadow-sm
`} />
```

This is a commonly used format where the content starts and ends on their own lines. The raw string length exceeds `printWidth` even though no individual line does — and the rule's own suggestion "consider splitting into multiple lines" can never produce a result that satisfies the rule.

The following test cases demonstrate this:

```ts
const veryLongClass =
  'flex items-center justify-between p-4 m-2 bg-white text-black rounded shadow-lg border w-full'

// Already split into two short lines — should be VALID, currently reports ✗
{
  code: `const className = \`\n  ${veryLongClass.slice(0, 40)}\n  ${veryLongClass.slice(40)}\n\``,
  filename: 'test.tsx',
  // Raw string: 100 chars (> 80), but longest individual line is 53 chars (< 80)
  // Rule currently reports: "Class string is 100 characters long, exceeding the print width of 80"
},

// Still on one long line — should remain INVALID ✓
{
  code: `const className = \`${veryLongClass}\``,
  filename: 'test.tsx',
  errors: [{ messageId: 'tooLong' }],
},

// Wrapped, but one line is itself still too long — should remain INVALID ✓
{
  code: `const className = \`\n  ${veryLongClass}\n  p-4\n\``,
  filename: 'test.tsx',
  errors: [{ messageId: 'tooLong' }],
},
```

The reported `length` in the error message is also wrong for multiline strings — it shows the raw total rather than the length of the offending line.

**Comment by sergioazoc** [comment]
Great catch, @fkoepke. You're right: `printWidth` was measuring the whole raw string, every newline
and indent included, so splitting across lines could never satisfy the rule. It even flagged its own
`classesPerLine` output as too long.

It now measures the longest individual line (and reports that length). A one-line string behaves
exactly as before. Fixed in #113, shipping in v1.8.0. Thanks for the clean before/after cases.
